documentation/src/main/asciidoc/reference/introduction.adoc

```diff
@@ -637,6 +639,28 @@ session.find(Book.class, id)
        .eventually(session::close);
 ----
 
+It's also important that the session obtained via `openSession`
+is used in the same Vert.x context it's been created or you might see the following exception:
+
+[source, JAVA, indent=0]
+----
+HR000068: This method should exclusively be invoked from a Vert.x EventLoop thread; ...
+----
+
+One way to run the code in the correct {vertx-context-introduction}[Vert.x context] is the following:
+
+[source, JAVA, indent=0]
+----
+Context currentContext = Vertx.currentContext();
+currentContext.runOnContext( event -> {
+   // Here you will be able to use the session
+});
+----
+
+Hibernate Reactive will create a default Vert.x instance when
+needed. If your application has one already, you can register it via the
+<<_vert_x_instance_service>>.
+ 
```

Comment:
I meant to move this part too. It makes more sense to discuss it in the section about the Vert.x instance.

It's really about _integration_, not so much about the programming model itself.